flasksrc/__init__.py

```diff
@@ -80,6 +81,10 @@ def create_app(test_config=None):
                      '/api/toynet/session/<int:toynet_session_id>/create/switch')
     api.add_resource(ToyNetSessionByIdCreateRouter,
                      '/api/toynet/session/<int:toynet_session_id>/create/router')
+    api.add_resource(ToyNetSessionByIdCreateLink,
+                     '/api/toynet/session/<int:toynet_session_id>/create/link')
+    api.add_resource(ToyNetSessionByIdDeleteLink,
+                     '/api/toynet/session/<int:toynet_session_id>/delete/link')
```

Comment:
Why not just make one endpoint `/api/toynet/session/<int:toynet_session_id>/link` and have it handle multiple HTTP methods, (i.e. DELETE, POST)?

Reply:
I wanted to keep the same convention as the devices where it is a `/[create|delete]/[name]` endpoint for consistency.